File: Fortran90/Unit01/Exercise05/src/exercise05.f90
program exercise05
    implicit none
    
    real :: a, b
    character(45) :: str

    ! Reading in the data
    open(1, file="../Unit01/Exercise05/Inputs/input.txt")
    read(1,*) str, a
    read(1,*) str, b
    close(1)

    ! Writing out the hypotenuse
    open(1, file="../Unit01/Exercise05/Outputs/output.txt")
    write(str,*) "c: ", sqrt((a*a) + (b*b))
    write(1,*) str
    close(1)

end program exercise05

File: Cpp/Unit01/Exercise05/src/exercise05.cpp
#include<iostream>
#include<fstream>
#include<string>
#include<cmath>

using namespace std;

int main() {
    float a, b;
    string str;

    // Reading in the values
    ifstream indata;
    indata.open("../Unit01/Exercise05/Inputs/input.txt");
    indata >> str >> a;
    indata >> str >> b;
    indata.close();

    // writing out the result
    ofstream outdata;
    outdata.open("../Unit01/Exercise05/Outputs/output.txt");
    outdata<<sqrt((a*a) + (b*b));
    outdata.close();

    return 0;
}